.github/files/setup-wordpress-env.sh

```diff
@@ -160,6 +160,16 @@ if [[ "$WITH_WOOCOMMERCE" == true ]]; then
 	echo "::endgroup::"
 fi
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+if [[ "$WITH_WPCOMSH" == true ]]; then
+	echo "::group::Installing wpcomsh into WordPress"
+
+	mkdir "/tmp/wordpress-$WP_BRANCH/src/wp-content/mu-plugins"
+	cp -r "/tmp/wordpress-$WP_BRANCH/src/wp-content/plugins/wpcomsh/src" "/tmp/wordpress-$WP_BRANCH/src/wp-content/mu-plugins/wpcomsh"
```

Comment:
This seems wrong. If we need to move it into mu-plugins at all, it should probably be like this
```suggestion
	cp -r "/tmp/wordpress-$WP_BRANCH/src/wp-content/plugins/wpcomsh" "/tmp/wordpress-$WP_BRANCH/src/wp-content/mu-plugins/wpcomsh"
```